src/main/java/hw_1/Task1.java

```diff
@@ -0,0 +1,13 @@
+package hw_1;
+
+public class Task1 {
+    public static void main(String[] args) {
+
+    String name = " Nataliya";
+    int age = 35;
+    float weight = 50.5f;
+    String result = name +" "+ age+" " + weight;
```

Comment:
Should be not "name +" "+ age+" " + weight;" but "name + " " + age + " " + weight";

Reply:
fixed